command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
I don't understand the leap here.  I see no reason why a single repo can't define two specs and someone can be compliant with one spec but not the other.  It we want the specs to be separate then we can do so. If we want them to be linked, or merged,then we can do so. We get to decide this stuff.